SAPHana/ra/SAPHanaController

```diff
@@ -349,6 +350,28 @@ function version() {
     fi
 }
 
+# function: get_action_timeout
+# output:  action timeout
+#
+# timeOut = max(3600, 95%(actionTimeOut))
+#
+function get_action_timeout() {
+    super_ocf_log info "FLOW $FUNCNAME ($*)"
+    # $OCF_RESKEY_CRM_meta_timeout is the timeout of the current running action in ms
+    local actionTimeOut="${OCF_RESKEY_CRM_meta_timeout:-$HANA_STD_ACTION_TIMEOUT}"
+    local stdTimeOut="$HANA_STD_ACTION_TIMEOUT"
+    local actTimeOutPercent=95 # 95% to left 5% for the rest of the resource agent action
+
+    # 95%(actionTimeOut)
+    ((timeout = actionTimeOut * actTimeOutPercent/1000))
```

Comment:
If actionTimeOut is in milliSeconds (OCF_RESKEY_CRM_meta_timeout) we need to divide by 1000.
For the percentage we need also to divide by 100. We need to decide, if actionTimeOut as input value is ment in seconds [s] or miliseconds [ms].

Reply:
right, fix is added to this PR. Thanks a lot for pointing this out.